src/utils/formatCurrencyAmount.ts

```diff
@@ -0,0 +1,31 @@
+import { isInteger } from 'lodash'
+import { V2V3CurrencyOption } from 'models/v2v3/currencyOption'
+import { V2V3_CURRENCY_ETH, V2V3_CURRENCY_USD } from './v2v3/currency'
+
+const formatAmount = (amount: number) => {
+  if (isInteger(amount)) {
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/isInteger Could we nix  lodash?